<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../../桌面/Pyslvs-PyQt5/icons.qrc">
    <normaloff>:/icons/collection-triangular-iteration.png</normaloff>:/icons/collection-triangular-iteration.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="Driver_label">
         <property name="text">
          <string>Drivers:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="Driver_list"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="Follower_label">
         <property name="text">
          <string>Followers:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="Follower_list"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="Target_label">
         <property name="text">
          <string>Targets:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="Target_list"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="constraint_label">
         <property name="text">
          <string>Gruebler's Equation:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="constraint_list"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Link_Expression_label">
       <property name="text">
        <string>Link expression:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Link_Expression">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Expression_label">
       <property name="text">
        <string>Expression:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Expression">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../桌面/Pyslvs-PyQt5/icons.qrc"/>
 </resources>
 <connections/>
</ui>
